x-pack/plugins/security_solution/public/timelines/components/open_timeline/helpers.ts

```diff
@@ -464,7 +465,7 @@ export const dispatchUpdateTimeline =
     }
 
     if (duplicate && ruleNote != null && !isEmpty(ruleNote)) {
-      const newNote = createNote({ newNote: ruleNote });
+      const newNote = createNote({ newNote: ruleNote, user: ruleAuthor || 'elastic' });
```

Comment:
I guess we should default to Elastic as that's what it does now, but wondering it's better to show like undefined or unknown or something, but then again may not even matter as it'll just show up as "U" ...

Reply:
theoretically this should never happen afaik, as if there is no user information in client side state, a user should just immediately be redirected to the login page I think, this is more of just a typescript bandaid, defaulting to elastic instead of empty string like we do in many other places

Reply:
👍🏾